src/models/expenses.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../db');
+
+const Expense = sequelize.define('Expense', {
+  id: {
+    type: DataTypes.INTEGER,
+    primaryKey: true,
+    autoIncrement: true,
+    allowNull: false,
+  },
+  userId: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  title: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  spentAt: {
+    type: DataTypes.DATE,
+    allowNull: false,
+  },
+  amount: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  category: {
```

Comment:
The 'category' field is defined as an array of strings, however, it's better to have a separate table for categories and create a relationship between the expense and category tables using foreign keys.